packages/core/src/styles/colors.scss

```diff
@@ -277,7 +277,7 @@ $color-map: ();
 					(
 						"#{$name}-#{$category}": $value
 					)
-				);
+				) !global;
```

Comment:
this makes the `color-map` a local variable, isn't it?
I think you may need to make `color-map` both global and local. 
when you make it only a local variable, our demo brokes.

Reply:
@sophiiae is there an approach that'll make both work? I haven't looked into it much, are you able to pick this up from here?

Reply:
> @sophiiae is there an approach that'll make both work? I haven't looked into it much, are you able to pick this up from here?

As I know so far, one global and one local `color-map` can make both work. 
But I'm still trying to find a better solution for this.

Reply:
Are you able to push what you're talking about into a PR? Not sure why demos are breaking, they're still just importing our SCSS styles...

Reply:
sure, I can make a PR for it.

Reply:
Thanks